Sources/ArcGISToolkit/Components/Offline/OfflineManager.swift

```diff
@@ -127,10 +127,7 @@ public class OfflineManager: ObservableObject {
     /// - Parameter portalItemID: The portal item ID.
     func removeMapInfo(for portalItemID: Item.ID) {
         offlineMapInfos.removeAll(where: { $0.portalItemID == portalItemID })
-        
-        let dir = URL.portalItemDirectory(forPortalItemID: portalItemID)
-        try? FileManager.default.removeItem(at: dir.appending(path: "info.json"))
```

Comment:
not good to hard-code these paths in so many places. I created an object to hold these so that they are only created once.